src/lisflood/hydrological_modules/reservoir.py

```diff
@@ -185,7 +185,7 @@ def dynamic_inloop(self, NoRoutingExecuted):
         option = settings.options
         maskinfo = MaskInfo.instance()
         if option['simulateReservoirs']:
-            InvDtSecDay = 1 / 86400.
+            InvDtSecDay = float(1/ 86400.)
```

Comment:
This is not needed since it's already a division between an integer and a float so it gives a float.
Also, please note that float(1/86400) = 0.0 and it does not solve this specific python2 issue.

Reply:
Please solve this in whatever way you find more appropriate.

Reply:
@ecCinziaMazzetti 
We can leave it as it is. Adding float there it's just redundant (so you can leave the float cast as well:))